indico/modules/users/__init__.py

```diff
@@ -39,6 +39,9 @@
     'use_markdown_for_minutes': False,
     'synced_fields': None,  # None to synchronize all fields, empty set to not synchronize
     'suggest_categories': False,  # whether the user should receive category suggestions
+    'mastodon_server_url': None,
+    'mastodon_server_name': None,
+    'mastodon_server_domain': None,
```

Comment:
Do we really need that? Seem like something you can just parse from the URL when needed.